Makefile

```diff
@@ -7,17 +7,22 @@ MONGO_SETTINGS=MONGO_CONNECTION=${MONGO_CONNECTION} \
 	CUSTOMCONNSTR_mongo_collection=${CUSTOMCONNSTR_mongo_collection} \
 	CUSTOMCONNSTR_mongo_settings_collection=${CUSTOMCONNSTR_mongo_settings_collection}
 
-.PHONY: all coverage test travis
+MOCHA=$(shell which mocha ./node_modules/mocha/bin/_mocha)
```

Comment:
Here I use `which name alt_name0 alt_name1 alt_name2`, which responds with the first executable path that exists.
This assigns the location of mocha, which may vary depending on environment, user preferences to a variable, and then re-uses the variable throughout.  Most tools that are installed locally are in `./node_modules/.bin/` on closer it look it looks like `mocha` does live there after all.

We can do same thing for `istanbul` and other tools, but `mocha` is